### ISSUE TITLE
COLMAP

### ISSUE THREAD

**Issue by sxj111** [nex-mpi/nex-code#4]
![image](https://user-images.githubusercontent.com/27821863/112593047-e622f080-8e41-11eb-8d30-98d77243a430.png)
When using the data generated by colmap to generate training, are these two npy files automatically generated by the training code ？`python train.py -scene ${PATH_TO_SCENE} -model_dir ${MODEL_TO_SAVE_CHECKPOINT} -http
`

**Comment by pureexe** [comment]
Yes, If you followed the instruction of [Using your own images.](https://github.com/nex-mpi/nex-code#using-your-own-images) 

The following code will generate `poses_bounds.npy` and `hwf_cxcy.npy`

https://github.com/nex-mpi/nex-code/blob/51ab4c154fd9a83f5213a33b7425b1272b52130c/utils/colmap_runner.py#L153-L154


--------

However, if you already have a COLMAP database and sparse model for your images. Then, you want to generate npy files without re-run COLMAP.

You have to comment or remove this line to use your own COLMAP data.

https://github.com/nex-mpi/nex-code/blob/51ab4c154fd9a83f5213a33b7425b1272b52130c/utils/colmap_runner.py#L190

**Comment by sxj111** [comment]
> Yes, If you followed the instruction of [Using your own images.](https://github.com/nex-mpi/nex-code#using-your-own-images)
> 
> The following code will generate `poses_bounds.npy` and `hwf_cxcy.npy`
> 
> https://github.com/nex-mpi/nex-code/blob/51ab4c154fd9a83f5213a33b7425b1272b52130c/utils/colmap_runner.py#L153-L154
> 
> However, if you already have a COLMAP database and sparse model for your images. Then, you want to generate npy files without re-run COLMAP.
> 
> You have to comment or remove this line to use your own COLMAP data.
> 
> https://github.com/nex-mpi/nex-code/blob/51ab4c154fd9a83f5213a33b7425b1272b52130c/utils/colmap_runner.py#L190

Thanks ，but I don't understand have a colmap database and spark model for your images. 
I shoot with my own camera, then run colmap, organize the data set according to the rules, and then train. I don't need to delete that line, do I?

**Comment by pureexe** [comment]
We provide an automatic COLMAP run in the training script for the user who doesn't get used to COLMAP. 

I recommend you run without removing anything first.

**Comment by sxj111** [comment]
> 我们为不习惯COLMAP的用户提供了在培训脚本中运行的自动COLMAP。
> 
> 我建议您先运行而不删除任何内容。

![image](https://user-images.githubusercontent.com/27821863/112962798-56e74700-9179-11eb-8069-fac9a52d2e1c.png)
I ran the result of colmap by myself, and then trained, I have commented out `runner(dpath) ,` but an error will be reported.
![image](https://user-images.githubusercontent.com/27821863/112963301-d1b06200-9179-11eb-8da0-9a5477bd9c7c.png)
![image](https://user-images.githubusercontent.com/27821863/112963348-db39ca00-9179-11eb-9211-07eeda33cd08.png)

**Comment by sxj111** [comment]
> We provide an automatic COLMAP run in the training script for the user who doesn't get used to COLMAP.
> 
> I recommend you run without removing anything first.

Thanks，I have solved the problem of COLMAP，but The result is a bit problematic.
I think it is a problem with planes.txt. Are there any tricks for setting the 4 parameters in this file?
![image](https://user-images.githubusercontent.com/27821863/113232746-53b29f00-92d0-11eb-84a8-963b7f3ee91d.png)
![image](https://user-images.githubusercontent.com/27821863/113232763-5dd49d80-92d0-11eb-8494-0dec514acb6f.png)
